src/PerfView/PerfViewData.cs

```diff
@@ -463,14 +463,24 @@ public static string ChangeExtension(string filePath, string newExtension)
             return Path.Combine(dirName, fileName);
         }
 
+        /// <summary>
+        /// for extensions of PerfView that don't have access to the GuiApp.MainWindow
+        /// </summary>
+        /// <param name="doAfter"></param>
+        public void Open(Action doAfter = null)
+        {
+            this.Open(GuiApp.MainWindow, GuiApp.MainWindow.StatusBar, doAfter);
+        }
+
         public override void Open(Window parentWindow, StatusBar worker, Action doAfter = null)
         {
             if (!m_opened)
             {
-                worker.StartWork("Opening " + Name, delegate ()
+                    worker.StartWork("Opening " + Name, delegate ()
```

Comment:
nit, I think the original spacing was correct.

Reply:
good catch